profiles/car.lua

```diff
@@ -158,7 +159,7 @@ local turn_bias                  = properties.left_hand_driving and 1/1.2 or 1.2
 local obey_oneway                = true
 local ignore_areas               = true
 local ignore_hov_ways            = true
-local ignore_toll_ways           = false
+local ignore_toll_ways = set_option("ignore_toll_ways", false)
```

Comment:
Looks like the `scripts/check-taginfo.py` does not like this ^ --- Travis failing.